assets/js/custom/UserNotifications.js

```diff
@@ -0,0 +1,152 @@
+      $(this).css('color', 'rgba(0,0,0,0.87)')
+      comment = true
+      document.getElementById('all-notif').style.backgroundColor = '#e0e0e0'
+      document.getElementById('all-notif').style.color = 'rgba(0,0,0,0.87)'
+      document.getElementById('follow-notif').style.backgroundColor = '#e0e0e0'
+      document.getElementById('follow-notif').style.color = 'rgba(0,0,0,0.87)'
+      document.getElementById('reaction-notif').style.backgroundColor = '#e0e0e0'
+      document.getElementById('reaction-notif').style.color = 'rgba(0,0,0,0.87)'
+      document.getElementById('remix-notif').style.backgroundColor = '#e0e0e0'
+      document.getElementById('remix-notif').style.color = 'rgba(0,0,0,0.87)'
+      all = false
+      follower = false
+      reactions = false
+      remixes = false
+    }
+  })
+  $(document).on('click', '#reaction-notif', function () {
+    if (reactions === false) {
+      $(this).css('background-color', '#acacac')
+      $(this).css('color', 'rgba(0,0,0,0.87)')
+      reactions = true
+      document.getElementById('all-notif').style.backgroundColor = '#e0e0e0'
+      document.getElementById('all-notif').style.color = 'rgba(0,0,0,0.87)'
+      document.getElementById('follow-notif').style.backgroundColor = '#e0e0e0'
+      document.getElementById('follow-notif').style.color = 'rgba(0,0,0,0.87)'
+      document.getElementById('comment-notif').style.backgroundColor = '#e0e0e0'
+      document.getElementById('comment-notif').style.color = 'rgba(0,0,0,0.87)'
+      document.getElementById('remix-notif').style.backgroundColor = '#e0e0e0'
+      document.getElementById('remix-notif').style.color = 'rgba(0,0,0,0.87)'
+      all = false
+      follower = false
+      comment = false
+      remixes = false
+    }
+  })
+  $(document).on('click', '#remix-notif', function () {
+    if (remixes === false) {
+      $(this).css('background-color', '#acacac')
+      $(this).css('color', 'rgba(0,0,0,0.87)')
+      remixes = true
+      document.getElementById('all-notif').style.backgroundColor = '#e0e0e0'
+      document.getElementById('all-notif').style.color = 'rgba(0,0,0,0.87)'
+      document.getElementById('follow-notif').style.backgroundColor = '#e0e0e0'
+      document.getElementById('follow-notif').style.color = 'rgba(0,0,0,0.87)'
+      document.getElementById('comment-notif').style.backgroundColor = '#e0e0e0'
+      document.getElementById('comment-notif').style.color = 'rgba(0,0,0,0.87)'
+      document.getElementById('reaction-notif').style.backgroundColor = '#e0e0e0'
+      document.getElementById('reaction-notif').style.color = 'rgba(0,0,0,0.87)'
+      all = false
+      follower = false
+      comment = false
+      reactions = false
+    }
+  })
+  self.markAsRead = function (notificationId, type, seen, id) {
+    if (!seen) {
+      $.ajax({
+        url: self.markAsReadUrl + '/' + notificationId,
+        type: 'get',
+        success: function (data) {
+          if (data.success) {
```

Comment:
success already is a 2XX response
there should be no reason to check a success or fail inside a success -  (Explained in the php route)